velox/exec/tests/NestedLoopJoinTest.cpp

```diff
@@ -24,26 +24,22 @@ using namespace facebook::velox::exec::test;
 
 class NestedLoopJoinTest : public HiveConnectorTestBase {
  protected:
-  RowTypePtr probeType_;
-  RowTypePtr buildType_;
-  std::vector<std::string> comparisons_;
-  std::vector<core::JoinType> joinTypes_;
-  std::string joinConditionStr_;
-  std::string queryStr_;
-
-  void SetUp() override {
-    HiveConnectorTestBase::SetUp();
-    probeType_ = ROW({{"t0", BIGINT()}});
-    buildType_ = ROW({{"u0", BIGINT()}});
-    comparisons_ = {"=", "<", "<=", "<>"};
-    joinTypes_ = {
-        core::JoinType::kInner,
-        core::JoinType::kLeft,
-        core::JoinType::kRight,
-        core::JoinType::kFull};
-    joinConditionStr_ = "t0 {} u0";
-    queryStr_ = "SELECT t0, u0 FROM t {} JOIN u ON t.t0 {} u.u0";
-  }
+  const std::string probeKeyName_{"t0"};
+  const std::string buildKeyName_{"u0"};
+  RowTypePtr probeType_{ROW({{probeKeyName_, BIGINT()}})};
```

Comment:
Can you move class members after class methods? Thanks!

Reply:
Done.